css/index.css

```diff
@@ -143,4 +237,133 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+header {
+    display: flex;
+    width: 880px;
+}
+
+.top-nav,
+bottom nav {
+    width: 810px;
+    display: flex;
+    justify-content: space-around;
+}
+
+.top-nav a {
+    display: flex;
+    margin: 10px;
+    border-radius: 5px;
+    padding: 15px;
+    justify-content: space-between;
+    color: black;
+    text-decoration: none;
+}
+
+.top-img {
+    padding: 10px;
+    margin-right: 10px;
+    margin-left: -25px;
+}
+
+.jumbo-img {
+    margin-top: 20px;
+}
+
+#box1 {
```

Comment:
this is fine. Are there any other ways you can write this by using `.box:nth-child(n)` so you don't have to use indivdual id's for each box?